include/lmms_math.h

```diff
@@ -72,30 +75,36 @@ inline auto fraction(std::floating_point auto x) noexcept
  * If the result is interpreted as a phase of an oscillator, it makes that negative phases are
  * converted to positive phases.
  */
-inline auto absFraction(std::floating_point auto x) noexcept
+inline auto absFraction(floating_point auto x) noexcept
 {
 	return x - std::floor(x);
 }
 
-inline auto fastRand() noexcept
+//! @brief Returns a psuedorandom integer between 0 and 32767, inclusive.
+inline integral auto fastRand() noexcept
 {
-	static unsigned long next = 1;
-	next = next * 1103515245 + 12345;
-	return next / 65536 % 32768;
+	thread_local unsigned long s_next = 1;
+	s_next = s_next * 1103515245 + 12345;
+	return s_next / 65536 % 32768;
 }
 
-template<std::floating_point T>
+
+//! @brief Returns a psuedorandom number between 0 and @p range, inclusive.
+template<typename T> requires is_arithmetic_v<T>
 inline auto fastRand(T range) noexcept
 {
-	constexpr T FAST_RAND_RATIO = static_cast<T>(1.0 / 32767);
+	constexpr auto FAST_RAND_RATIO = static_cast<conditional_t<is_floating_point_v<T>, T, float>>(1.0 / 32767);
 	return fastRand() * range * FAST_RAND_RATIO;
 }
 
-template<std::floating_point T>
-inline auto fastRand(T from, T to) noexcept
-{
-	return from + fastRand(to - from);
-}
+
+//! @brief Returns a psuedorandom number between @p from and @p to, inclusive.
+inline auto fastRand(auto from, auto to) noexcept {	return from + fastRand(to - from); }
```

Comment:
```suggestion
auto fastRand(auto from, auto to) noexcept { return from + fastRand(to - from); }
```
Remove tab also.

> Would you clarify your reasoning for suggesting this?

In the past, I was told to remove `inline`, since it is implicit.